src/PKSim.R/Domain/ParameterValueWithUnit.cs

```diff
@@ -0,0 +1,36 @@
+﻿using OSPSuite.Core.Domain.Populations;
+using PKSim.Core.Model;
+
+namespace PKSim.R.Domain
+{
+   public class ParameterValueWithUnit
```

Comment:
Just a wrapper to export unit alongside parameter path and values. This is something that was requested quite a few times for the R export